coverage-report/README.md

```diff
@@ -0,0 +1,16 @@
+## Coverage Report
+
+This module is meant to gather aggregate jacoco test coverage metrics across the `gax-java` and `showcase` modules.
+
+### Unit Test Coverage
+In order to view aggregate unit test coverage of GAX in both `gax-java` and `showcase`:
+
+1. At the root of the repository, run `mvn clean test -DenableTestCoverage`.
+2. The metrics can be found at `${HOME}/gapic-generator-java/coverage-report/target/site/jacoco-aggregate/index.html`
```

Comment:
I believe the home directory is irrelevant?

Reply:
Agreed, starting with `gapic-generator-java` should be sufficient.